### ISSUE TITLE
Elevated circulating reverse T3 concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7576]
**Preferred term label:**
Elevated circulating reverse T3 concentration

**Synonyms**
Elevated circulating reverse triiodothyronine concentration
Elevated rT3

**Definition (free text, please give PubMed ID)**

Reverse T3 (3,3′,5′-triiodothyronine or rT3) is the third most abundant iodothyronine circulating in human blood and is produced by the inner ring deiodination of the pro-hormone thyroxine (T4).  As rT3 binds weakly to the T3 thyroid nuclear hormone receptors, it is thought to represent an inactive end-product of thyroid hormone metabolism, diverting T4 away from T3 production. Serum rT3 concentration is increased as part of the non-thyroidal illness syndrome, and by administration of common medications such as amiodarone which inhibit the metabolism of rT3. Serum rT3 concentration is also affected by genetic conditions that affect the iodothyronine deiodinases, as well as thyroid transporters and transport proteins.
The number 4 in T4 means it has 4 iodine atoms. It can lose 1 of them, yielding either T3 or reverse T3, depending on which iodine atom it loses ([Figure 3](https://www.ccjm.org/content/85/6/450#F3)). Loss of iodine from the five-prime (5′) position on its outer ring yields T3, the most potent thyroid hormone, produced at a rate of 30 to 40 µg per day.[21](https://www.ccjm.org/content/85/6/450#ref-21) On the other hand, when T4 loses an iodine atom from the five (5) position on its inner ring it yields reverse T3, produced at a rate slightly less than that of T3, 28 to 40 µg per day.[21](https://www.ccjm.org/content/85/6/450#ref-21) Reverse T3 is inactive.

PMID: 29883303 

**Parent term (use hpo.jax.org/app)**
Abnormal thyroid hormone level HP:0031508

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619855 
PMID: 32718224  8/8

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also  Elevated serum rT3/T3 ratio
619855
PMID: 32718224 8/8

**Comment by pnrobinson** [comment]
added
